app/models/showdate/mailchimp_mailer.rb

```diff
@@ -0,0 +1,101 @@
+require 'digest'
+
+class MailchimpMailer
+  def initialize(key = nil)
+    @apikey = key || (Option.mailchimp_key rescue nil)
+    @disabled = !!@apikey.blank?
+    @errors = []
```

Comment:
Should we call `mailchimp_init` in initialization instead? On one hand we would be sending a request on every initialization of the object, but on the other hand, you can't do anything without Mailchimp init